index.html

```diff
@@ -17,21 +17,28 @@
                 <a href="#" class="logo">
                     <img class="menuIcone" src="./Images/logo.png" alt="logo Image">
                 </a>
-                <button class="hambug-menu">
-                    <i class="menuIcon material-icons"> <img class="menuIcone" src="./Images/union.png"
-                            alt="Union Image"></i>
-                </button>
-                <ul class="mobile-menu">
-                    <li><a class="menuItem" href="#Portfolio">Portfolio</a></li>
-                    <li><a class="menuItem" href="#About">About</a></li>
-                    <li><a class="menuItem" href="#contact">Contacts</a></li>
-                </ul>
+                <!--button class="hambug-menu">
+                    <i class="menuIcon material-icons">
+                        <img class="menuIcone" src="./Images/union.png" alt="Union Image">
+                    </i>
+                </button-->
+               <input type="checkbox" class="side-menu" id="side-menu"/>
+               <label for="side-menu" class="hamb">
+                <span class="hamb-line"></span>
+               </label>
+                <div class="nav">
+                    <ul class="menu">
+                        <li><a href="#Portfolio">Portfolio</a></li>
+                        <li><a href="#About">About</a></li>
+                        <li><a href="#contact">Contacts</a></li>
+                    </ul>
+                </div>
 
-                <ul class="nav-links">
+                <!--ul class="nav-links">
                     <li class="links">Portfolio</li>
                     <li class="links">About</li>
                     <li class="links">Contact</li>
-                </ul>
+                </ul-->
```

Comment:
- [ ] You should think on removing unused code as it makes your code unclean and not proffessional